exec-analysis/src/main/java/com/capitalone/dashboard/exec/model/Person.java

```diff
@@ -0,0 +1,178 @@
+    }
+
+    public void setDistinguishedName(String distinguishedName) {
+        this.distinguishedName = distinguishedName;
+    }
+
+    public String getObjectCategory() {
+        return objectCategory;
+    }
+
+    public void setObjectCategory(String objectCategory) {
+        this.objectCategory = objectCategory;
+    }
+
+    @Attribute(name = "displayName")
+    private String displayName;
+
+    @Attribute(name = "directReports")
+    private Set<String> directReports;
+
+    @Attribute(name = "thumbnailPhoto")
+    private byte[] thumbnailPhoto;
+
+    public Set<String> getDirectReports() {
+        return directReports;
+    }
+
+    public void setDirectReports(Set<String> directReports) {
+        this.directReports = directReports;
+    }
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public void setDisplayName(String displayName) {
+        this.displayName = displayName;
+    }
+
+    public byte[] getThumbnailPhoto() {
+        return thumbnailPhoto;
+    }
+
+    public void setThumbnailPhoto(byte[] thumbnailPhoto) {
+        this.thumbnailPhoto = thumbnailPhoto;
+    }
+
+    public String getName() {
+        return this.displayName;
+    }
+
+    public Person() {
+    }
+
+    public Person(String fullName, String lastName) {
+        this.ntid = fullName;
+        this.lastName = lastName;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Name getDn() {
+        return dn;
+    }
+
+    public void setDn(Name dn) {
+        this.dn = dn;
+    }
+
+    public String getNtid() {
+        return ntid;
+    }
+
+    public void setNtid(String ntid) {
+        this.ntid = ntid;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getManager() {
+        if (manager != null) {
+            return manager;
+        } else {
+            return "";
+        }
+    }
+
+    public void setManager(String manager) {
+        this.manager = manager;
+    }
+
+    /**
+     * Method to get manager's NTID
+     *
+     * @return manager NTID
+     */
+    public String getManagerNTID() {
+        return ((!getManager().isEmpty()) ? getManager().split(",")[0].split("=")[1] : "");
+    }
+
+    /**
+     * Method to get all direct reporting person NTID
+     *
+     * @return List of NTID
+     */
+    public List<String> getReportingNTIDs() {
+        return (directReports != null)
+                ? directReports.stream().map(entry -> entry.split(",")[0].split("=")[1]).collect(Collectors.toList())
+                : new ArrayList<String>();
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" + "dn=" + dn + ", " + ", NTID='" + ntid + '\'' + ", lastName='" + lastName + '\'' + "," + '}';
```

Comment:
We prefer the usage of `org.apache.commons.lang3.builder.ToStringBuilder` here

Reply:
Converted from toString to ToStringBuilder as suggested.